monkey/agent_plugins/exploiters/smb/src/smb_client.py

```diff
@@ -0,0 +1,297 @@
+import logging
+from dataclasses import dataclass
+from io import BytesIO
+from pathlib import PureWindowsPath
+from typing import Any, Dict, Optional, Sequence, Tuple, Type
+
+from impacket.dcerpc.v5 import scmr, srvs, transport
+from impacket.dcerpc.v5.rpcrt import DCERPC_v5
+from impacket.smbconnection import SMB_DIALECT, SessionError, SMBConnection
+
+from common.credentials import Credentials, LMHash, NTHash, Password, Secret, get_plaintext
+from common.types import NetworkPort
+from infection_monkey.i_puppet import TargetHost
+
+logger = logging.getLogger(__name__)
+
+ERROR_SERVICE_REQUEST_TIMEOUT = 1053
```

Comment:
This is not a timeout, it's an error code
```suggestion
SERVICE_REQUEST_TIMEOUT_ERROR = 1053
```